src/common/libsdexec/start.c

```diff
@@ -301,6 +314,15 @@ static int prop_add (json_t *prop, const char *name, const char *val)
                 return -1;
         }
     }
+    else if (streq (name, "OOMScoreAdjust")) {
+        errno = 0;
+        char *endptr;
+        long l = strtol (val, &endptr, 10);
+        if (errno != 0 || *endptr != '\0')
+            return -1;
+        if (prop_add_i32 (prop, name, l) < 0)
+            return -1;
+    }
```

Comment:
Unrelated question: would it work to usethe json type as a hint so that we don't have to explicitly add support to the code for each new, non-string systemd property? If the signedness of number is a problem, maybe an optional type hint could be supported in the string? Just something to think about for the future since systemd seems to have _a lot_ of these properties and it is a bummer if we have to support each in this way.

On topic: I think the range of OOM scores is `-1000` to `1000`, should that be checked here? Or is there a sensible enough error when a user uses a value outside that range already?